src/wp-includes/class-wp-duotone.php

```diff
@@ -920,16 +921,7 @@ private static function enqueue_global_styles_preset( $filter_id, $duotone_selec
 	 * @param WP_Block_Type $block_type Block Type.
 	 */
 	public static function register_duotone_support( $block_type ) {
-		$has_duotone_support = false;
-		if ( property_exists( $block_type, 'supports' ) ) {
-			/*
-			 * Previous `color.__experimentalDuotone` support flag is migrated
-			 * to `filter.duotone` via `block_type_metadata_settings` filter.
-			 */
```

Comment:
I'd like to have this comment retained to help direct folks looking for where `__experimentalDuotone` is handled.